.github/workflows/deploy.yml

```diff
@@ -8,6 +8,7 @@ env:
   CARGO_NET_RETRY: 10
   RUST_BACKTRACE: short
   RUSTUP_MAX_RETRIES: 10
+  MACOSX_DEPLOYMENT_TARGET: 10.7
```

Comment:
Just in case, please also add this to `workflow.yml`.

Reply:
Done.